### ISSUE TITLE
Checkout: Make "WordPress.com Credits" payment summary less confusing and more useful

### ISSUE THREAD

**Issue by DavidRothstein** [Automattic/wp-calypso#88863]
### What

1. Start with an account that has sufficient WordPress.com Credits to cover an entire purchase.
2. Put a renewal in your shopping cart (note this won't work for a new purchase because https://github.com/Automattic/wp-calypso/pull/79029 made it so that the "WordPress.com Credits" payment method doesn't show in that case).
3. Notice the summary just says "WordPress.com Credits":

![wpcom-credits-summary](https://github.com/Automattic/wp-calypso/assets/235183/c17ed8ba-42c0-4462-8e30-c9abe845bd05)

4. If you click "Edit", you can get more information (including the amount of credits you have available) but it's strange to have to click "Edit" to do that, especially since there's only one payment method available and therefore nothing to edit:

![wpcom-credits-full](https://github.com/Automattic/wp-calypso/assets/235183/419d916d-b99d-4eac-9d02-9ed6ab95a6be)

Overall, seems like the amount of credits could be shown in the summary view also (easy fix?) and ideally there wouldn't be an "Edit" button in the first place if the payment method can't be edited (perhaps a more general issue).

### Why

To make checkout easier to understand.  Discussed in p1711381869380079/1711348128.936479-slack-C02BEP610 also.

**Comment by DavidRothstein** [comment]
Just realized this behavior may be a side effect of auto-collapsing the payment method section (https://github.com/Automattic/wp-calypso/pull/83985), since before then the full version presumably was shown when checkout was loaded (without ever having an "Edit" link to click first).

A fix for this (or at least a partial fix) may be easy, so I'm adding the "Quick fix" label also.

**Comment by sirbrillig** [comment]
Agreed. This is because prior to collapsing that step, the "inactive" state (`inactiveContent`) for each payment method was mostly ignored since it was never displayed. Here's the component that needs to be edited to show more info:

https://github.com/Automattic/wp-calypso/blob/b78c4acdc30a3d65cd84b513aac1796cbbf8277c/client/my-sites/checkout/src/payment-methods/free-purchase.tsx#L135

Maybe we can change it to be similar to the active content:

https://github.com/Automattic/wp-calypso/blob/b78c4acdc30a3d65cd84b513aac1796cbbf8277c/client/my-sites/checkout/src/payment-methods/free-purchase.tsx#L108

**Comment by michaeldcain** [comment]
Since this is a minor enhancement and a `Quick Fix`, I'm moving this to the backlog for a future Quick Fix day.
